pkg/modules/AWS/eks/locals.tf

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Please ignore this change, also change in `eks/output.tf`. Those are related to another PR.